docs/cloud/pouta/command-line-tools.md

```diff
@@ -33,12 +33,12 @@ that uses the common openstack command line tool.
     python-novaclient==9.0.1
     python-swiftclient==3.3.0
 
-##### Openstack Commands and help
+##### Openstack commands and help
 
     openstack -h
 
-You can get a list of  available openstack subcommands by appending -h
-after openstack command. If you wish to see the options for a specific
+You can get a list of the available OpenStack subcommands by appending -h
+after the OpenStack command. If you wish to see the options for a specific
```

Comment:
The second mention is the cli tool : openstack. The first is the project name, so second should be lower case or then highlighted.